style.css

```diff
@@ -106,3 +98,7 @@ transition: 0.5s all ease; -moz-transition:0.5s all ease, -webkit-transition:0.5
     from    { -webkit-transform:translateX(200%); }
     to      { -webkit-transform: translateX(0); }
 }
+
```

Comment:
check your diff before committing. There shouldn't be any new lines at the end of a file (besides the obligatory 1 that your text editor should put, but won't appear).